tests/python_test.py

```diff
@@ -284,3 +284,11 @@ def test_write_resource_invalid_align_sector(
     with pytest.raises(ValueError):
         sanlock.write_resource(
             "ls_name", "res_name", disks, align=align, sector=sector)
+
+
+def test_write_resource_invalid_path(tmpdir, sanlock_daemon):
+    path = str(tmpdir.join("resources"))
+    disks = [path]
+    # test parsing resource which is not a list of tuples
+    with pytest.raises(ValueError):
+        sanlock.write_resource("ls_name", "res_name", disks)
```

Comment:
We want to check that the error message includes our bad input:

    assert repr("not a tuple") in e.value

Finally, we need to check that this handles correctly non-asci unicode and encoded strings. We can use parameters to run this tests with many bad values.

Also note that a newline is missing at the end of the file.